api/ccapi/include/tensor_dim.h

```diff
@@ -43,27 +48,32 @@ class TensorDim {
   /**
    * @brief Construct a new Tensor Dim object
    *
+   * @param fm format NCHW | HNWC
    * @param eff_dim_flag_ effective dimension flag (1 means it's effective)
    * @param dyn_dim_flag_ dynamic dimension flag (1 means it's unspecified)
    */
-  explicit TensorDim(const std::bitset<MAXDIM> &eff_dim_flag_ = 0b1111,
+  explicit TensorDim(Format fm = Format::NCHW,
```

Comment:
If we move the `Format` to last order, we can assume it's NCHW when it's not provided as other APIs.

Reply:
yes. but there is also a need for ```eff_dim_flag_``` and ```dyn_dim_flag_```. so.. if we set the TensorDim with Format.. we have to define these flags as well. I think it is quite bothering.

Reply:
OK. `eff_dim_flag_` and `dyn_dim_flag_` are not usually changed, original one would be better.